src/components/MarketMyTrades/index.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react'
+import Decimal from 'decimal.js'
+
+// import { calcLMSRMarginalPrice, calcLMSROutcomeTokenCount } from 'api'
+
+import DecimalValue from 'components/DecimalValue'
+import CurrencyName, { collateralTokenToText } from 'components/CurrencyName'
+import { COLOR_SCHEME_DEFAULT } from 'utils/constants'
+
+import './marketMyTrades.less'
+
+class MarketMyTrades extends Component {
+
+  componentWillMount() {
+    const { market, defaultAccount } = this.props    
+    if (!this.props.trades || this.props.trades.length == 0) {
+      // Retrieve participant trades to state
+      this.props.fetchMarketParticipantTrades(market.address, defaultAccount)
+    }    
+  }
+
+  getAverageCost(order) {
+    if (order.orderType == 'BUY') {
+      return order.cost/1e18
+    }
+    else if (order.orderType == 'SELL') {
+      return order.profit/1e18
+    }
+    else if (order.orderType == 'SHORT SELL') {
+      return order.cost/1e18
+    }    
```

Comment:
this should be a linter error, I think you need to explicitly return a value if none of these cases are true